.github/workflows/deploy-cpp.yml

```diff
@@ -175,10 +22,14 @@ jobs:
       name: "pypi"
       url: "https://pypi.org/project/awkward-cpp/"
     steps:
+
     - uses: actions/download-artifact@v4
       with:
-        pattern: "awkward*"
+        pattern: "awkward-cpp*"
         path: dist
         merge-multiple: true
 
+    - name: List distributions to be deployed
+      run: ls -l dist/
```

Comment:
This doesn't do anything beyond `ls` all the files that are getting published out into the logs so that they can be inspected after the fact if desired.

If you ever wanted to debug part of this workflow this now means that on your debug branch you can just comment out the `uses: pypa/gh-action-pypi-publish` bit below and then you can just look at the logs when you run to see if all the files you expect are indeed there.

Reply:
Okay. (I do a lot of debugging by `ls`, anyway.